libs/sdk-core/src/lib/auth/auth.ts

```diff
@@ -26,7 +26,9 @@ export class Authentication implements IAuthentication {
   }
 
   getAuthHeaders(): { [key: string]: string } {
-    return this.nuclia.options.apiKey
+    return this.nuclia.options.standalone
+      ? { 'X-NUCLIADB-ROLES': 'READER' }
+      : this.nuclia.options.apiKey
       ? { 'X-STF-Serviceaccount': `Bearer ${this.nuclia.options.apiKey}` }
       : this.getToken()
       ? { Authorization: `Bearer ${this.getToken()}` }
```

Comment:
FYI I remember from my previous job this kind of imbrication of conditional ternary operator was raising ESlint error: https://eslint.org/docs/latest/rules/no-nested-ternary#rule-details
That being said, having it multiline like this makes it quite understandable to me here.